lib/benchee/formatters/console.ex

```diff
@@ -79,29 +90,31 @@ defmodule Benchee.Formatters.Console do
     |> to_string
   end
 
-  defp comparison_report([_reference]) do
+  defp comparison_report([_reference], _) do
     [] # No need for a comparison when only one benchmark was run
   end
-  defp comparison_report([reference | other_jobs]) do
-    report = [reference_report(reference) | comparisons(reference, other_jobs)]
-    [comparison_descriptor | report]
+  defp comparison_report([reference | other_jobs], label_width) do
+    [
+      comparison_descriptor(), reference_report(reference, label_width) |
+      comparisons(reference, label_width, other_jobs)
```

Comment:
I think splitting it like this is a bit hard to read (e.g. after the `|`) - I'd either reintroduce the local variable or start the new line after the `,`